src/entities/streaming/model/slice.ts

```diff
@@ -1,7 +1,7 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 
-const DEFAULT_HOST = '127.0.0.1';
-const DEFAULT_PORT = '8881';
+const DEFAULT_HOST = '';
+const DEFAULT_PORT = '';
```

Comment:
Please wait for the reply to my comment here: https://mindlogger.atlassian.net/browse/M2-3512?focusedCommentId=13919
If I am right then the changes should be reverted